README.md

```diff
@@ -1,6 +1,6 @@
 [![DOI](https://zenodo.org/badge/131810578.svg)](https://zenodo.org/badge/latestdoi/131810578)
 [![codecov](https://codecov.io/github/GEOS-DEV/GEOS/graph/badge.svg?token=0VTEHPQG58)](https://codecov.io/github/GEOS-DEV/GEOS)
-![CI](https://github.com/GEOS-DEV/GEOS/actions/workflows/ci_tests.yml/badge.svg)
+[![CI](https://github.com/GEOS-DEV/GEOS/actions/workflows/ci_tests.yml/badge.svg)](https://github.com/GEOS-DEV/GEOS/actions?query=branch%3Adevelop)
```

Comment:
```suggestion
[![CI](https://github.com/GEOS-DEV/GEOS/actions/workflows/ci_tests.yml/badge.svg)](https://github.com/GEOS-DEV/GEOS/actions/workflows/ci_tests.yml)
```
I think this shows the workflow corresponding to the badge status

Reply:
that seems to point to the latest CI job that was run which can be on any PR related branch. I think it is more useful to point to the latest job that was run for `develop`

Reply:
ok but does the badge know about that?

Reply:
```suggestion
[![CI](https://github.com/GEOS-DEV/GEOS/actions/workflows/ci_tests.yml/badge.svg?branch=develop)](https://github.com/GEOS-DEV/GEOS/actions?query=branch%3Adevelop)
```
I guess like this it does